tests/test_06_jws.py

```diff
@@ -1020,3 +1026,50 @@ def test_verify_json_missing_key():
 
     # With both
     assert JWS().verify_json(_jwt, keys=[vkeys[0], sym_key])
+
+
+def test_is_compact_jws():
+    _header = {"foo": "bar", "alg": "HS384"}
+    _payload = "hello world"
+    _sym_key = SYMKey(key=b"My hollow echo chamber", alg="HS384")
+
+    _jwt = JWS(msg=_payload, alg='HS384').sign_compact(keys=[_sym_key])
```

Comment:
Can we get a non-compact JWS fed into `is_compact_jws` as well?

Reply:
Since this PR gives us `is_json_jws` and `is_compact_jws` I thought about adding `is_jws` containing something like this:

```
def is_jws(token):
   if is_json_jws(token):
      return 'json'
  elif is_compact_jws(token):
     return 'compact'
  return False
```

Reply:
Yes, sounds fair.